go-app/report/analyzeLog.go

```diff
@@ -160,7 +162,8 @@ func getIssueDetails(cfgFile *Config, fContent string, headerMap map[string]bool
 						if err != nil {
 							continue
 						}
-						proc_issue = strings.Join(proc_rgx_comp.FindAllString(fContent, -1), "\n")
+						raw_proc_issue := proc_rgx_comp.FindAllString(fContent, -1)
+						proc_issue = strings.Join(raw_proc_issue, "\n")
```

Comment:
The change here is a little unnecessary because we don't need raw_proc_issue for anything else.

Reply:
I made this change because last time you said that the line is too long